crates/downloader/src/main.rs

```diff
@@ -1347,6 +1354,30 @@ async fn download(
     }
 }
 
+// FIXME: HTTPステータスを確認したりSHA-256をチェックしたりといったまともな方法にする。
+// ただしどちらもoctocrab側の対応が必要。
+// cf. https://github.com/VOICEVOX/voicevox_core/issues/1120
+/// [`octocrab::repo::ReleasesHandler::stream_asset`]でダウンロードしたものを検証する。
+fn validate_github_asset_content(content: Vec<u8>, kind: GhAssetKind) -> anyhow::Result<Vec<u8>> {
+    match (kind, &*content) {
+        (GhAssetKind::Archive(ArchiveKind::Zip), [0x50, 0x4b, 0x03, 0x04, ..])
+        | (GhAssetKind::Archive(ArchiveKind::Tgz), [0x1f, 0x8b, 0x08, ..]) => Ok(content),
+        (_, content) => Err({
+            let mut msg = "予期しない応答をGitHubが返しました".to_owned();
+            if let Ok(content) = str::from_utf8(content) {
+                msg += ": ";
+                msg += content.trim_end();
+                if content.contains("API rate limit exceeded for") {
+                    msg += " (note: レートリミットによるエラーのようです。\
+                            認証トークンを設定することでレートリミットは緩和されます。\
+                            詳細は`--help`をご覧ください)";
```

Comment:
一般的には`Note:`らしい？
あと日本語リファクタリングしてみました。
日本語ドキュメントをリファクタリングしてくれるAI用のプロンプトいつか考えたいですねぇ。

```suggestion
                    msg += " (Note: レートリミットによるエラーの可能性があります。\
                            認証トークンを設定すると制限が緩和されます。\
                            詳細は`--help`をご覧ください)";
```